src/main.py

```
@@ -0,0 +1,35 @@
+import asyncio
+import uvicorn
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+from watchdog.observers.polling import PollingObserver
+from config import *
```

Ruff (0.11.9)

`from config import *` used; unable to detect undefined names

(F403)